packages/contracts-bedrock/test/L2/ETHLiquidity.t.sol

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+// Error imports
+import "src/libraries/errors/CommonErrors.sol";
+
+/// @title ETHLiquidity_Test
+/// @notice Contract for testing the ETHLiquidity contract.
+contract ETHLiquidity_Test is CommonTest {
+    /// @notice Emitted when an address burns ETH liquidity.
+    event LiquidityBurned(address indexed caller, uint256 value);
+
+    /// @notice Emitted when an address mints ETH liquidity.
+    event LiquidityMinted(address indexed caller, uint256 value);
+
+    /// @notice The starting balance of the ETHLiquidity contract.
+    uint256 public constant STARTING_LIQUIDITY_BALANCE = type(uint248).max;
+
+    /// @notice Test setup.
+    function setUp() public virtual override {
+        super.enableInterop();
+        super.setUp();
+    }
+
```

Comment:
It could be good to have a test that asserts the balance is `STARTING_LIQUIDITY_BALANCE` - this is kind of like a constructor test